install/first_google_boot.sh

```diff
@@ -233,6 +235,35 @@ function extract_spinnaker_kube_credentials() {
   fi
 }
 
+function extract_spinnaker_gcr_credentials() {
+  local config_path="$GCR_DIR/gcr.json"
+  mkdir -p $(dirname $config_path)
+  chown -R spinnaker:spinnaker $(dirname $config_path)
+
+  if clear_metadata_to_file "gcr_json" $config_path; then
+    # This is a workaround for difficulties using the Google Deployment Manager
```

Comment:
I'd rather keep it (or a reference to this comment elsewhere) since I'm hopefully adding first class GCR & Kubernetes support to the DM marketplace deployer soon and will get to see firsthand if this works